lib/mix/phoenix.ex

```diff
@@ -109,18 +110,20 @@ defmodule Mix.Phoenix do
         {_, _} -> false
        end)
     |> Enum.into(%{}, fn
-        {k, {:array, _}}      -> {k, []}
-        {k, :integer}         -> {k, 42}
-        {k, :float}           -> {k, "120.5"}
-        {k, :decimal}         -> {k, "120.5"}
-        {k, :boolean}         -> {k, true}
-        {k, :map}             -> {k, %{}}
-        {k, :text}            -> {k, "some content"}
-        {k, :date}            -> {k, "2010-04-17"}
-        {k, :time}            -> {k, "14:00:00"}
-        {k, :datetime}        -> {k, "2010-04-17 14:00:00"}
-        {k, :uuid}            -> {k, "7488a646-e31f-11e4-aace-600308960662"}
-        {k, _}                -> {k, "some content"}
+        {k, {:array, _}}         -> {k, []}
+        {k, :integer}            -> {k, 42}
+        {k, :float}              -> {k, "120.5"}
+        {k, :decimal}            -> {k, "120.5"}
+        {k, :boolean}            -> {k, true}
+        {k, :map}                -> {k, %{}}
+        {k, :text}               -> {k, "some content"}
+        {k, :date}               -> {k, "2010-04-17"}
+        {k, :time}               -> {k, "14:00:00"}
+        {k, :datetime}           -> {k, "2010-04-17 14:00:00"}
+        {k, :uuid}               -> {k, "7488a646-e31f-11e4-aace-600308960662"}
+        {k, {:unique, :integer}} -> {k, 42}
```

Comment:
Good point, stupid oversight from my end. I think that should work, the only change is that it might be easier to work with {t, :unique} if I change the task to work with attr:type:unique.